.github/workflows/k8-infra-ci.yaml

```diff
@@ -64,4 +66,29 @@ jobs:
         quiet: true
         soft_fail: true
         output_format: json
-    
\ No newline at end of file
+
+  release-charts:
+    name: Release Charts
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
```

Comment:
I will also add the condition below, so release should only run on the success of helm-chart-validation.
needs: helm-chart-validation

Reply:
Good point and it makes sense, added this in the latest commit.